pkg/helm/controller.go

```diff
@@ -174,7 +205,8 @@ func job(chart *helmv1.HelmChart) (*batch.Job, *core.ConfigMap, *core.ConfigMap)
 		Spec: batch.JobSpec{
 			BackoffLimit: &oneThousand,
 			Template: core.PodTemplateSpec{
-				ObjectMeta: meta.ObjectMeta{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{},
```

Comment:
why did you have to add the annotation map here?

Reply:
I added an empty Annotations map here so that I can add the values hash key to it later, after we've generated ConfigMaps.